# The systemd log reader should be replaced with another solution
# For now we disable it
--- a/endpoints/system/logs.py
+++ b/endpoints/system/logs.py
@@ -11,7 +11,6 @@
 import psutil
 
 from tools.config import Config
-from tools.logs import LogReader
 from tools.permissions import SystemAdminROPermission
 
 
@@ -32,7 +31,7 @@
     n = int(request.args.get("n", 10))
     skip = int(request.args.get("skip", 0))
     after = datetime.strptime(request.args["after"], "%Y-%m-%d %H:%M:%S.%f") if "after" in request.args else None
-    return jsonify(data=LogReader.tail(log.get("format", "journald"), log["source"], n, skip, after))
+    return jsonify(message="Log file not found"), 404
 
 
 @API.route(api.BaseRoute+"/system/updateLog/<int:pid>", methods=["GET"])
